makeflow/src/makeflow.c

```diff
@@ -754,6 +761,11 @@ static int makeflow_check(struct dag *d)
 			if(skip_file_check || batch_fs_stat(remote_queue, f->filename, &buf) >= 0) {
 				continue;
 			}
+
+			if(f->source) {
```

Comment:
Currently, I do not suppose the `--mounts` and `-c` options should be used concurrently.
However, I may be too strict on how the user uses these options.
Let us give it more discussion.